src/main/java/ch/njol/skript/lang/function/Namespace.java

```diff
@@ -50,46 +51,44 @@ public enum Origin {
 	public static class Key {
 		
 		private final Origin origin;
-		
-		private final String name;
 
-		public Key(Origin origin, String name) {
+		@Nullable
+		private final String script;
+
+		public Key(Origin origin, @Nullable String script) {
 			super();
 			this.origin = origin;
-			this.name = name;
+			this.script = script;
 		}
 		
 		public Origin getOrigin() {
 			return origin;
 		}
-		
-		public String getName() {
-			return name;
+
+		@Nullable
+		public String getScript() {
```

Comment:
getScriptName maybe? Also is the old method not used anywhere?

Reply:
Oh right, I always forget about backwards compatibility 😭 I'll revert the name changes

Reply:
It wasn't used, atleast by Skript.